.github/workflows/release.yml

```diff
@@ -1,45 +1,99 @@
 on:
   push:
-    # Sequence of patterns matched against refs/tags
-    tags:
-      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+    branches:
+      # On develop, we build and publish containers, with the tag of "develop-latest"
+
+      # Ex. A newly build coingecko adapter is built and pushed to ECR.
+      # The ECR registry is reachable at public.ecr.aws/z0b1w9r9/adapters/
+
+      # You would be able to pull the coingecko adapter with the following command:
+      # docker pull public.ecr.aws/z0b1w9r9/adapters/coingecko-adapter:develop-latest
+      - develop
+
+      # On master, we build and publish containers with the tag of "latest" and another tag of the
+      # adapters version contained within package.json.
+
+      # Ex. If the value of the field "version" in the "package.json" file of the coingecko adapter is "0.0.5-beta",
+      # The "coingecko-adapter" container will be resolvable with the following specifiers:
+      # coingecko-adapter:latest
+      # coingecko-adapter:0.0.5-beta
+
+      # The ECR registry is reachable at public.ecr.aws/z0b1w9r9/adapters/
+
+      # You would be able to pull the coingecko adapter with the following commands:
+      # docker pull public.ecr.aws/z0b1w9r9/adapters/coingecko-adapter:latest
```

Comment:
Should be safe to replace `z0b1w9r9` with `chainlink`—it works the same and the alias is properly claimed.